free-courses-en.md

```diff
@@ -583,6 +584,14 @@
 * [Ruby Tutorial](https://www.w3resource.com/ruby/)
 * [RubyMonk - Interactive Ruby tutorials](https://rubymonk.com)
 
+### Rust
+
+* [Intro to Rust](https://www.youtube.com/playlist?list=PLJbE2Yu2zumDF6BX6_RdPisRVHgzV02NW) - Tensor Programming (YouTube)
+* [Rust Basics](https://www.youtube.com/playlist?list=PLlcnQQJK8SUjApd95LIcd3K9XXmE-IeCS) - Engineer Man (YouTube)
+* [Rust Crash Course](https://www.youtube.com/watch?v=zF34dRivLOw) - Traversy Media (YouTube 1.5hr)
+* [Rust Projects](https://www.youtube.com/playlist?list=PLJbE2Yu2zumDD5vy2BuSHvFZU0a6RDmgb) - Tensor Programming (YouTube)
+* [Rust Tutorial](https://www.youtube.com/playlist?list=PLLqEtX6ql2EyPAZ1M2_C0GgVd4A-_L4_5) - Doug Milford (YouTube)
+
```

Comment:
Hello! Thank you for the resources

Currently, the build is failing due to not having an extra blank line after the last resource in the Ruby section and before your Rust heading.

Check the reason why [here](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)

Also, no need to mention (Youtube), check that [here](https://github.com/EbookFoundation/free-programming-books/issues/4579#issuecomment-716214451)


```suggestion
* [Ruby Tutorial](https://www.w3resource.com/ruby/)
* [RubyMonk - Interactive Ruby tutorials](https://rubymonk.com)


### Rust

* [Intro to Rust](https://www.youtube.com/playlist?list=PLJbE2Yu2zumDF6BX6_RdPisRVHgzV02NW) - Tensor Programming
* [Rust Basics](https://www.youtube.com/playlist?list=PLlcnQQJK8SUjApd95LIcd3K9XXmE-IeCS) - Engineer Man
* [Rust Crash Course](https://www.youtube.com/watch?v=zF34dRivLOw) - Traversy Media
* [Rust Projects](https://www.youtube.com/playlist?list=PLJbE2Yu2zumDD5vy2BuSHvFZU0a6RDmgb) - Tensor Programming
* [Rust Tutorial](https://www.youtube.com/playlist?list=PLLqEtX6ql2EyPAZ1M2_C0GgVd4A-_L4_5) - Doug Milford

```

Reply:
You also didn't need to remove that blank line in your recent commit, that was fine before!

Reply:
Thanks kadhirash. I was getting confused about that blank lines as I didn't pay attention.

Reply:
Np, anytime!